src/lib/email-scrubber.js

```diff
@@ -0,0 +1,213 @@
+/* flow */
+/* eslint-disable no-console, no-param-reassign, no-mixed-spaces-and-tabs */
+
+import dns from 'dns';
+import { Socket } from 'net';
+import EventEmitter from 'events';
+import winston from 'winston';
+import callbackToPromise from './promisify';
```

Comment:
don't name it  as `callbackToPromise`, it's not converting a callback to a promise, it's converting a node callback based API to promise based API.